modules/monitoring-contents-of-the-metrics-ui.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="contents-of-the-metrics-ui_{context}"]
+= Contents of the Metrics UI
+
+This section shows and explains the contents of the Metrics UI, a Web interface to Prometheus.
+
+The *Metrics* page is located in *Monitoring* -> *Metrics* of the {product-title} web console.
+
+image::../images/metrics-screen.png[]
```

Comment:
Does it make sense to disambiguate this filename? Some (but not all) of the images, such as for ossm and olm, use a prefix for the filename.

Reply:
Good idea, implemented across all monitoring images.